docs/doc-loader.php

```diff
@@ -77,7 +77,8 @@ public static function fromPath($path) {
         $outputPath = __DIR__ . '/../demo/' . $path . '.expect';
 
         assert(file_exists($codePath), "code does not exist $codePath");
-        assert(file_exists($outputPath), "output does not exist $outputPath");
+        // allow demos without expect files - we will just get a warning later
+        // assert(file_exists($outputPath), "output does not exist $outputPath");
```

Comment:
Why does this need to be removed? Perhaps we can skip and print a warning for now if the output path does not exist?

Reply:
In PHP 7 this generated a warning and the tests still passed. With PHP 8 this generates a fatal error and stops.

A warning is still generated later when `file_get_contents` is called, but the tests still work.